scripts/kubernetes/sth-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
```

Comment:
Maybe this should be in `./docs` not in `./scripts`?

Reply:
seems good to me, thanks for commit :)